packages/gatsby/cache-dir/query-result-store.js

```diff
@@ -115,42 +115,37 @@ export class PageQueryStore extends React.Component {
   }
 }
 
-export class StaticQueryStore extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      staticQueryData: { ...getStaticQueryResults() },
-    }
-  }
+export const StaticQueryStore = ({ children }) => {
```

Comment:
This gives the component a name during develop

```suggestion
export const StaticQueryStore = function StaticQueryStore({ children }) {
```